### PR TITLE
Issue-11412 Add link following Overview image

### PR DESCRIPTION
**Pull request by bmcelvee**
Fixes https://github.com/openshift/openshift-docs/issues/11412

### CODE REVIEW

**Comment by bmcelvee** [comment]
@adellape PTAL :cake:

File: architecture/index.adoc [hunk 1]
```diff
@@ -44,6 +44,8 @@ as they flow through your system
 
 .{product-title} Architecture Overview
 image::architecture_overview.png[{product-title} Architecture Overview]
+
+For more information on the nodes in the architecture overview see xref:infrastructure_components/kubernetes_infrastructure.adoc#node[Nodes].
```
Comment (adellape) [inline architecture/index.adoc]: To also capture "master nodes" in this pointer, maybe the following tweak, linking to the top of the page:

"For more information on the node types in the architecture overview, see xref:infrastructure_components/kubernetes_infrastructure.adoc#architecture-infrastructure-components-kubernetes-infrastructure[Kubernetes Infrastructure]."

**Review by adellape — COMMENTED** [review]
Minor suggestion, but LGTM!

**Comment by bmcelvee** [comment]
Minor change (adding link) - no QE required.

**Pull request merged**